packages/react/spec/auto/PolarisAutoForm.spec.tsx

```diff
@@ -701,22 +716,24 @@ function loadMockWidgetCreateMetadata() {
   });
 }
 
-function loadMockWidgetUpdateMetadata() {
+function loadMockWidgetUpdateMetadata(mockFindByResult = true) {
   expect(mockUrqlClient.executeQuery.mock.calls[0][0].variables).toEqual({
     modelApiIdentifier: "widget",
     modelNamespace: null,
     action: "update",
     includeRelatedFields: false,
   });
 
-  mockUrqlClient.executeQuery.pushResponse("widget", {
-    stale: false,
-    hasNext: false,
-    data: getWidgetRecord({
-      name: "Test Widget",
-      inventoryCount: 42,
-    }),
-  });
+  if (mockFindByResult) {
```

Comment:
I think this should be a separate helper function rather than a boolean flag

Reply:
Updated such that this is now a separate helper function